static/atom.less

```diff
@@ -28,3 +28,4 @@
 @import "text";
 @import "utilities";
 @import "octicons";
+@import "white-cursor";
```

Comment:
How about renaming it to just `cursor`? There might be a chance that we add more custom cursors that would go into the same file.